sklearn/svm/tests/test_bounds.py

```diff
@@ -74,3 +76,41 @@ def test_ill_posed_min_c():
 def test_unsupported_loss():
     with pytest.raises(ValueError):
         l1_min_c(dense_X, Y1, loss='l1')
+
+
+# TODO test deterministic results on differing systems
+# def test_set_seed():
+#     def _test(seed, val):
+#         # TODO currently causes 99% coverage in test_bounds.py
+#         if seed is not None:
+#             set_seed_wrap(seed)
+#         x = bounded_rand_int_wrap(100)
+#         assert(x == val), 'Expected {} but got {} instead'.format(val, x)
+
+#     # TODO should be default seed for std::mt19937,
+#     # but different results on jupyter lab
+#     # _test(None, 81)
+#     _test(5489, 81)  # default seed for std::mt19937
+#     _test(0, 54)
+#     _test(4294967295, 9)  # max unsigned int size
+
+
+_MAX_INT = 2147483647
+
+
+@pytest.mark.parametrize("orig_range", [_MAX_INT, 100])
+def test_bounded_rand_int(orig_range):
+    n_pts = 1000
```

Comment:
Maybe it would be good to also parametrize the number of points, as `_MAX_INT` is, well, a bit large. Something like this ?

```suggestion
@pytest.mark.parametrize("orig_range,n_pts", [(_MAX_INT, 10000), (100, 10)])
def test_bounded_rand_int(orig_range, n_pts):
```